.github/workflows/build.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - main
+    - drcandacemakedamoore/docker
```

Comment:
does this branch need to stay, or was it added for testing purposes?

Reply:
I think this is a good idea to leave for now, in case we want to run actions on a rebased version. If you recall the docker run needs to be doubled so it runs in a 3.10 and 3.11 docker; and that will take a bit of work.